sigstore/_utils.py

```diff
@@ -178,7 +178,7 @@ def read_embedded(name: str, prefix: str) -> bytes:
     Read a resource embedded in this distribution of sigstore-python,
     returning its contents as bytes.
     """
-    return resources.files("sigstore._store").joinpath(prefix, name).read_bytes()  # type: ignore
+    return resources.files("sigstore._store").joinpath(prefix, name).read_bytes()
```

Comment:
this was failing the `make lint`, so I removed it

Reply:
> this was failing the make lint, so I removed it

Sorry, that's my fault -- the CI needs it, because we lint in CI against our minimum supported Python. So please add that back in 🙂